### PR TITLE
Nemo 7765/consistent identification

### PR DESCRIPTION
**Pull request by sramalho94**
### Ticket(s)

_Required unless this is just a change to documentation._

- [7765](https://airtable.com/appfJZShN8K4tcWHU/pagXdnDYi0tVl4u8R?HjEAY=recgE4FgLQDMPqabj)

### Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

### Description

This task involved adding aria-labels to several elements on the website, including:

   - Social Media Icons
   - Access Calculator Button
   - Get Started Button
   - Learn More Button
   
To achieve this, modifications have been made to the respective page, type, content, and layout files in order to incorporate the necessary aria-label data.

### Screenshots

No changes to website visuals, the only changes were the additions of aria-labels 

### Checklist:

_Please delete any options that are not relevant_

- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings in the console
- [X] There are no new linting errors/problems in my code
- [X] I have filled this PR out fully, and cleaned up any extraneous items so that it is clear and easy to understand

### CODE REVIEW

File: pages/get-involved.tsx [hunk 1]
```diff
@@ -119,6 +119,11 @@ function Partner() {
                 popup={false}
                 setShareLinkCopied={() => {}}
                 shareLinkCopied={false}
+                ariaLabels={{
```
[the remark this answers is not in the document]
Comment (e-papanicolas) [reply pages/get-involved.tsx:122]: Or alternatively into the get involved content file?

**Review by e-papanicolas — CHANGES_REQUESTED** [review]
@sramalho94 One tiny change and then we are good. Really happy to see all of this accessibility stuff getting the attention it deserves.

File: components/layout/GetStartedStep.tsx [hunk 2]
```diff
@@ -19,6 +20,7 @@ export default function GetStartedStep({
   ctaText,
   ctaLink,
   children,
+  ariaLabel,
```
Comment (LeoFuna) [inline components/layout/GetStartedStep.tsx:23]:
```suggestion
  ariaLabels,
```
Can i give a little suggestion here? 
Thinking about legibility, when we just say `ariaLabel` we cant say from what, right?
Could be from Title, SectionContainer, etc.

So what do you think about, insted of using `ariaLabel`, use `ariaLabels` as an Object type which inside define from which one.

At this case could be something like `type ariaLabels = { ctaButton: string }`
Basically the same you did at `ShareButtons`.

There are other places in this PR that we can use the same concept to change.
Let me know if this make any sense :smiley:

File: pages/get-involved.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (sramalho94) [reply pages/get-involved.tsx]: Fixed, I just directly applied the aria-labels in the ShareButtons component

File: components/layout/GetStartedStep.tsx (hunk 2 above)
Comment (sramalho94) [reply components/layout/GetStartedStep.tsx]: that's a great suggestion @LeoFuna! I think I've applied it to all the parts of the code where this applies. Please let me know if I missed anything. There are some elements where I just directly apply the aria-labels, not through props, and those remain unchanged

Comment (LeoFuna) [reply components/layout/GetStartedStep.tsx]: Seems pretty good! Loved it 👏

File: components/layout/GetStartedStep.tsx [hunk 3]
```diff
@@ -4,13 +4,15 @@ import {
...
 
+import { AriaLabels } from '../../content/content.types.ts';
 import SectionContainer from './SectionContainer.tsx';
 
 interface GetStartedStepProps extends PaperProps {
   title?: string;
   bodyText?: string;
   ctaText?: string;
   ctaLink?: string;
+  ariaLabels?:AriaLabels;
```
Comment (e-papanicolas) [inline components/layout/GetStartedStep.tsx:15]: Will you just add a space after the colon here and ...

File: components/layout/PaperSection.tsx [hunk 4]
```diff
@@ -3,11 +3,14 @@ import {
...
 import React from 'react';
 
+import { AriaLabels } from '../../content/content.types.ts';
+
 interface PaperSectionProps extends PaperProps {
   title?: string;
   subtitle?: string;
   ctaText?: string;
   ctaLink?: string;
+  ariaLabels?:AriaLabels;
```
Comment (e-papanicolas) [inline components/layout/PaperSection.tsx:13]: here!

File: content/content.types.ts [hunk 5]
```diff
@@ -1,3 +1,6 @@
+export interface AriaLabels {
+    ctaButton?: string
+}
```
Comment (e-papanicolas) [inline content/content.types.ts:3]: And an extra line after this.

**Review by e-papanicolas — CHANGES_REQUESTED** [review]
@sramalho94 tiny proofreading corrections.

File: content/content.types.ts (hunk 5 above)
Comment (sramalho94) [reply content/content.types.ts:3]: @e-papanicolas oooff sorry my sloppiness, corrections have been implemented

Comment (e-papanicolas) [reply content/content.types.ts:3]: All good!

**Review by e-papanicolas — APPROVED** [review]

**Pull request merged**